Cold-Gas-Thruster/Cold Gas Thruster PDD.tex

```diff
@@ -0,0 +1,462 @@
+Dr. Patru would certainly be a great resource for this kind of help.
+
+The basic coding abilites include writing a MATLAB script to analyze and plot data. This is not hard for anyone who has coded before. If it proves to be a roadblock
+one can look up the MATLAB scripts already in the google drive for reference. These scripts layout all required techniques and a bit more, such as an iterative approach.
+
+The machining will be the most difficult to learn if there are no MEs or METs that have taken the proper courses. Machining can be extremely dangerous and cannot be done
+without the proper safety training. If no one is trained on the machines one can ask a machinist to make simple parts or submit a job at the Brickmann lab, formula machine shop.
+They have professional engineers there that can make nearly anything one could need. But their timeline slips often, so leave an appropriate amount of time for them to make the part
+so it does not interfere with the projects timeline.
+
+\subsection{Funding Requirements}
+  % Estimate costs that would be needed to meet objectives.
+Fortunately, most of the materials for the test stand have been purchased. This eliminates much of the system cost. The most costly part of the project
+will be the propulsion system (the regulator, solenoid, nozzle, piping, and interfaces). A preliminary, worst-case, scenario was determined by finding most of the parts
+on McMaster Carr. The total cost was approximately \$450. That would be the last part of the system needed before testing can occur. Accounting for unforseen cost then rounding upwards,
+the consquentical estimated cost is put at \$600 for the entirty of the project. A bills of materials is displayed in Appendix B.
+
+\subsection{Faculty Support}
+  % Identify faculty that will be involved (or would need to be involved) to meet objectives.
+  % Note that if a professor is the Principal Investigator (P.I.) for a project, there still needs to be a student as the SPEX Project Champion.
+Very limited faculty support will be necessary. The SPEX propulsion team has taken care of most of the parts that will need the assistance from faculty.
+This included the flow system and how it operates. Is it safe and is it necessary? Those questions were answered by Dr. Garrick of the MET department.
+There may be some assistance required if the nozzle performance if far below nominal, but guessing from past work it will not be a problem. There may also
+be the need for assistance during the data analysis phase or if the team chooses to perform flow simulations. But it will be a small amount of help.
+
+For the data analysis, if assistance is required for fully understanding it, Dr. Olles would be a good professor to ask. He has a PhD in rocket science
+so he could certainly help. Also, a SPEX member, Varun, has experience in flow simulation so he could probably understand the data sufficiently too.
```

Comment:
Do not specify individual members. PDDs should be timeless, in that they do not depend strongly on the current state of SPEX to be understood.